.codecov.yml

```diff
@@ -0,0 +1,16 @@
+coverage:
```

Comment:
Could you add a comment for the docs for these options, or add a comment line-by-line for what each of these does?

Reply:
Oh, you did it.